restler/restler_settings.py

```diff
@@ -747,6 +791,17 @@ def include_req():
         else:
             return include_req() and not exclude_req()
 
+    @property 
+    def token_authentication_method(self):
+        ## TODO: Verify OneOf?
+        if self.token_module_file:
```

Comment:
Yes, could you please fail if there is more than one method specified?

Reply:
Actually could you please add this in 'validateOptions'?

Reply:
adding this to validate_auth_options() as suggested in other comment